lib/pf/web/externalportal.pm

```diff
@@ -34,6 +35,12 @@ use pf::web::constants;
 use pf::web::util;
 use pf::constants;
 
+# Some vendors don't support some charatcters in their redirect URL
+# This here below allows to map some URLs to a specific switch module
+Readonly our $SWITCH_REWRITE_MAP => {
+    'RuckusSmartZone' => 'Ruckus::SmartZone',
```

Comment:
No, we want to the URL /RuckusSmartZone to the name of the module (Ruckus::SmartZone) since you can't put `:` in the redirect URL in the SmartZone itself